app/javascript/src/Navigation/components/ContextSwitcherContainer.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react'
```

Comment:
The idea is to have the Context Selector in a different file, no matter if it's small, because is not part of the others PF4 components (Brand, Toolbar, Avatar). And because is possible that we will refactor this in the future.

Reply:
I don't get it, what difference it makes?
```jsx
const Brand = () => (
  <div className="u-header-brand">
    <div className="u-header-left-half">
      <div className='Header-logo'>Red Hat 3scale API Management</div>
    </div>
    <div className="u-header-right-half">
      <ContextSwitcher />
      <ContextSelector />
    </div>
  </div>
)
```
Also I just noticed we're braking our own React architecture here... we are rendering a React Header that should contain a ContextSelector. Instead we are hard-rendering it directly by using React.render(). It should be a composition not a collage, if you know what I mean.

Reply:
I agree with Josemi, this component is off. Having our ContextSelector component we need to use is it as expected.

Reply:
Refactoring